openedge.properties

```diff
@@ -2,12 +2,18 @@ category=Languages
 description=Code Analyzer for OpenEdge
 homepageUrl=https://github.com/Riverside-Software/sonar-openedge
 archivedVersions=
-publicVersions=2.13.1
+publicVersions=2.13.1,2.14.4
 defaults.mavenGroupId=eu.rssw.sonar.openedge
 defaults.mavenArtifactId=sonar-openedge-plugin
 
+2.14.4.description=New expression engine in ABL Parser
+2.14.4.sqVersions=[7.9,9.1]
```

Comment:
7.9 is fully past EOL. This range needs to start with 8.9.

Additionally, are you sure you don't want to end the range with `LATEST`? 9.2.1 is already out, and this range excludes those users.

Reply:
Wouldn't that prevent SQ7.9 users from upgrading to the latest version of the plugin?
The range end was set to 9.1 because the plugin had been tested up to this version. My understanding was that plugins should be tested with the latest releases before upgrading this version number, but that may create a lot of PRs in this repo.  I've updated this PR to use LATEST.

Reply:
7.9 is fully past EOL. We do not support it any more. any where. That means here too. 7.9 users don't get plugin upgrades. They get to upgrade SonarQube itself.